samples/04_gis_analysts_data_scientists/classifying_human_activity_using _tabPFN_classifier.ipynb

```diff
@@ -0,0 +1,2181 @@
+{
```

Comment:
<p>an extra slash in home</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2189/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>removed</p>